openpype/modules/webserver/webserver_cli.py

```diff
@@ -0,0 +1,151 @@
+import time
```

Comment:
Both `webpublish_routes.py` and `webserver_cli.py` is webpublisher specific code. Should not be inside webserver module but in `webpublisher` host.

Reply:
Makes sense.